fuel-tx/src/tests/valid_cases/transaction/blob.rs

```diff
@@ -69,6 +70,22 @@ fn check__fails_if_maturity_not_met() {
     assert_eq!(Err(ValidityError::TransactionMaturity), result);
 }
 
+#[test]
+fn check__fails_if_expiration_not_met() {
+    // Given
+    let block_height: BlockHeight = 1000.into();
+    let failing_block_height = block_height.pred().unwrap();
+    let tx = valid_blob_transaction()
+        .expiration(failing_block_height)
+        .finalize_as_transaction();
+
+    // When
+    let result = tx.check(block_height, &test_params());
+
+    // Then
+    assert_eq!(Err(ValidityError::TransactionExpiration), result);
```

Comment:
Why do we check the happy and sad paths for `script` and `create`, but not for `blob`, `upgrade`, and `upload`?

Reply:
It follows the same pattern as the others policies, the correct cas is only tested in `valid_cases.rs` file.

Reply:
Okeyyyy, added :)